### PR TITLE
using null_count in isnull extractor

### PR DESCRIPTION
**Pull request by artemru**
* adding some shortcut for isna by using null_count attribute
* adding more tests
linked to #3

### CODE REVIEW

File: tests/test_pandas_integration.py [hunk 1]
```diff
@@ -131,13 +138,11 @@ def test_nbytes():
     assert array.nbytes >= 8
 
 
-@pytest.mark.xfail
 def test_series_attributes():
     s = pd.Series(fr.FletcherArray(TEST_ARRAY))
     assert s.ndim == 1
     assert s.size == 3
     assert s.values is not None
-    # This line currently fails with pandas master: https://github.com/pandas-dev/pandas/issues/22414
```
Comment (artemru) [inline tests/test_pandas_integration.py]: that was fixed !

**Comment by artemru** [comment]
we could also cache, `_null_mask`, if we don't allow setitem.

File: fletcher/_algorithms.py [hunk 2]
```diff
@@ -47,19 +47,19 @@ def extract_isnull_bytemap(chunked_array):
...
+    if chunked_array.null_count == 0:
+        return result
 
     offset = 0
     for chunk in chunked_array.chunks:
-        valid_bitmap = chunk.buffers()[0]
-        if valid_bitmap:
-            # TODO(ARROW-2664): We only need to following line to support
-            #   executing the code in disabled-JIT mode.
-            buf = memoryview(valid_bitmap)
```
Comment (krivonogov) [inline fletcher/_algorithms.py]: what was a point to do this casting before?

File: tests/test_pandas_integration.py [hunk 3]
```diff
@@ -22,10 +22,31 @@
...
-        chunks.append(pa.array(TEST_LIST))
-    return pa.chunked_array(chunks)
+    return pa.chunked_array([pa.array(TEST_LIST) for _ in range(10)])
+
+
+@pytest.fixture(
+    params=["all", "all_float", "some_in_all_chunks", "only_in_some_chunk", "none"]
+)
+def test_array_chunked_nulls(request):
+    case_dict = {
```
Comment (krivonogov) [inline tests/test_pandas_integration.py]: should we define this dict somewhere outside of this test? it can be useful for other tests

Comment (artemru) [reply tests/test_pandas_integration.py]: you can reuse a fixture ?

File: fletcher/_algorithms.py (hunk 2 above)
Comment (artemru) [reply fletcher/_algorithms.py]: there was no iteration implemented on buffer see ARROW-2664.

**Comment by artemru** [comment]
I've added the test you required, it's green - merging.

**Pull request merged**